packages/theme/src/cli/commands/theme/push.ts

```diff
@@ -135,11 +121,6 @@ export default class Push extends ThemeCommand {
   async run(): Promise<void> {
     const {flags} = await this.parse(Push)
 
-    if (flags.password || flags.legacy) {
```

Comment:
[Validate \`Push\` command output in CI/CD workflows · Issue #337 · Shopify/develop-advanced-edits](https://github.com/Shopify/develop-advanced-edits/issues/337)